src/main/java/graphql/execution/SimpleDataFetcherExceptionHandler.java

```diff
@@ -0,0 +1,25 @@
+package graphql.execution;
+
+import graphql.ExceptionWhileDataFetching;
+import graphql.language.SourceLocation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The standard handling of data fetcher error involves placing a {@link ExceptionWhileDataFetching} error
+ * into the error collection
+ */
+public class SimpleDataFetcherExceptionHandler implements DataFetcherExceptionHandler {
+    private static final Logger log = LoggerFactory.getLogger(SimpleDataFetcherExceptionHandler.class);
+
+    @Override
+    public void accept(DataFetcherExceptionHandlerParameters handlerParameters) {
+        Exception exception = handlerParameters.getException();
+        SourceLocation sourceLocation = handlerParameters.getField().getSourceLocation();
+        ExecutionPath path = handlerParameters.getPath();
+
+        ExceptionWhileDataFetching error = new ExceptionWhileDataFetching(path, exception, sourceLocation);
+        handlerParameters.getExecutionContext().addError(error);
+        log.warn(error.getMessage(), exception);
```

Comment:
I can see your point regarding logging.  For those that want highly tailored logging, having libraries do logging is painful.

But the opposite is true.  For those how just want it to function out of the box, then sprinkling logging helps them

I think a different PR should consider the logging aspects of the library